### PR TITLE
Handled an issue that authentication failure causes the settings page stuck in loading

### PR DESCRIPTION
**Pull request by pengchensfdc**

### CODE REVIEW

File: trunk/includes/pardot-api-class.php [hunk 1]
```diff
@@ -533,11 +535,12 @@ function get_response($item_type, $args = [], $property = 'result', $paged = 1)
 
 		} elseif (wp_remote_retrieve_response_code($http_response) >= 400 && wp_remote_retrieve_response_code($http_response) <= 499) {
 			$response = new SimpleXMLElement(wp_remote_retrieve_body($http_response));
-			if ($response->children()->err->attributes()->code == 184) {
+			if ($response->children()->err->attributes()->code == 184 && $retries < 3) {
+                $retries += 1;
```
Comment (kyungkimsf) [inline trunk/includes/pardot-api-class.php:539]: indentation is off here

File: trunk/README.txt [hunk 2]
```diff
@@ -158,6 +158,10 @@ Filter the regular expression used to find URLs to be converted to https://go.pa
 
 == Changelog ==
 
+= 1.5.8 =
+
+* Fix - Handled an issue that authentication failure causes the settings page stuck in loading
```
Comment (kyungkimsf) [inline trunk/README.txt:163]: would you mind linking to the 184 error code page so we are being clear on exactly what auth failure

File: README.md [hunk 3]
```diff
@@ -163,6 +163,9 @@ add_filter( 'pardot_https_regex', 'pardot_custom_filter_https_regex' );
 
 ## Changelog
 
+### 1.5.8
+* Fix - Handled an issue that authentication failure ([Pardot API Error Code 184](https://developer.salesforce.com/docs/marketing/pardot/guide/error-codes.html#numerical-list-of-error-codes)) causes the settings page stuck in loading
```
Comment (akeaswaransfdc) [inline README.md:167]: Nit: "causes the settings page to be stuck in a loading state"

Comment (kyungkimsf) [reply README.md:167]: @pengchensfdc please update the wording here, otherwise, it looks good.

**Review by kyungkimsf — APPROVED** [review]

**Pull request merged**